src/data/data.js

```diff
@@ -199,5 +199,16 @@ export const data = [
     { "id": 197, "sequence": ["G3", "Bb3", "C4", "C4", "Eb4", "D4", "Bb3", "Eb4", "F4", "Eb4", "D4", "C4", "D4", "C4", "Bb3", "G3"], "duration":[2.8, 2.8, 2.8, 1.6, 1.6, 2.8, 0.8, 1.6, 1.6, 2.8, 2.8, 1.6, 1.6, 2.8, 3.6, 5.6], "song":"The Planets, Op. 32 - Jupiter, the Bringer of Jollity (Gustav Holst)", "key":{"note":"Eb", "major":true}, "hasFlats":true},
     { "id": 198, "sequence": ["G4", "C5", "G4", "F4", "Eb4", "Eb4", "D4", "D4", "D4", "G4", "F4", "Eb4", "D4", "Eb4", "C4", "C4", "G4", "C5", "G4", "F4", "Eb4", "Eb4", "D4", "D4", "G4", "F4", "Eb4", "D4", "C4", "C4"], "duration":[1, 1.5, 0.5, 1, 1, 1, 1, 1, 1, 1.5, 0.5, 1, 1, 1, 1, 1, 1, 1.5, 0.5, 1, 1, 1, 1, 2, 1.5, 0.5, 1, 1, 2, 2], "song":"Säkkijärven polkka (Traditional Finnish Folk)", "key":{"note":"Eb", "major":true}, "hasFlats":true},
     { "id": 199, "sequence": ["G3", "Bb3", "Bb3", "C4", "Bb3", "D4", "G4", "F4", "D4", "Bb3", "C4", "Bb3", "D4", "C4", "Bb3", "G3", "G3", "D4", "D4", "C4", "Bb3", "G3"], "duration":[1.4, 1.4, 1.4, 1.4, 1.4, 9.8, 1.4, 4.2, 1.4, 4.2, 1.4, 1.4, 4.2, 1.4, 1.4, 4.2, 1.4, 4.2, 1.4, 1.4, 1.4, 4.2], "song":"My Body is a Cage (Peter Gabriel)", "key":{"note":"G", "major":false}, "hasFlats":true},
-    { "id": 200, "sequence": ["F#4","B4", "F#4", "B4", "D5", "B4", "D5", "B4", "D5", "F#5", "D5", "F#5", "D5", "F#5", "A5", "A4", "D5", "A4", "D5", "F#5"], "duration":[0.4, 1.2, 0.4, 0.8, 2.4, 2.4, 1.2, 0.4, 0.8, 2.4, 2.4, 1.2, 0.4, 0.8, 2.4, 2.4, 1.2, 0.4, 0.8, 4.8], "song":"Ride of the Valkyries (Richard Wagner)", "key":{"note":"D", "major":true}, "hasFlats":false}
+    { "id": 200, "sequence": ["F#4","B4", "F#4", "B4", "D5", "B4", "D5", "B4", "D5", "F#5", "D5", "F#5", "D5", "F#5", "A5", "A4", "D5", "A4", "D5", "F#5"], "duration":[0.4, 1.2, 0.4, 0.8, 2.4, 2.4, 1.2, 0.4, 0.8, 2.4, 2.4, 1.2, 0.4, 0.8, 2.4, 2.4, 1.2, 0.4, 0.8, 4.8], "song":"Ride of the Valkyries (Richard Wagner)", "key":{"note":"D", "major":true}, "hasFlats":false},
+    { "id": 201, "sequence": ["G4", "E5", "G4", "C5", "E5", "G#4", "A4", "A4", "A#4", "B4", "E5", "G4", "F#4", "F4", "C#5", "C5", "E4"], "duration":[6.3, 2.1, 1.4, 1.4, 0.7, 10.5, 5.6, 0.7, 0.7, 0.7, 0.7, 0.7, 0.7, 0.7, 0.7, 0.7, 4.9], "song":"Take the \"A\" Train (Billy Strayhorn)", "key":{"note":"C", "major":true}, "hasFlats":false},
```

Comment:
```suggestion
    { "id": 201, "sequence": ["G4", "E5", "G4", "C5", "E5", "G#4", "A4", "A4", "A#4", "B4", "E5", "G4", "F#4", "F4", "C#5", "C5", "E4"], "duration":[6.3, 2.1, 1.4, 1.4, 0.7, 10.5, 5.6, 0.7, 0.7, 0.7, 0.7, 0.7, 0.7, 0.7, 0.7, 0.7, 4.9], "song":"Take the 'A' Train (Billy Strayhorn)", "key":{"note":"C", "major":true}, "hasFlats":false},
```

Reply:
Did you test that in the UI the modal doesn't include the "\"? when someone plays the song?